components/sessionModals/CountDown.js

```diff
@@ -71,8 +41,9 @@ const Countdown = (props) => {
         <div className="flex-auto">
           <p className="mr-6">{props.bannerContent[0]}</p>
           <p className="font-bold mr-6">
-            {ready &&
-              `${props.bannerContent[1]} ${clock.minutes} ${props.bannerMinutesAnd} ${clock.seconds} ${props.bannerSeconds}.`}
+            {props.minutes || props.seconds
+              ? `${props.bannerContent[1]} ${props.xminutes} ${props.bannerMinutesAnd} ${props.xseconds} ${props.bannerSeconds}.`
```

Comment:
```suggestion
              ? `${props.bannerContent[1]} ${props.minutes} ${props.bannerMinutesAnd} ${props.seconds} ${props.bannerSeconds}.`
```
Looks like `xminutes` should just be `minutes` based on `Countdown.js`, same with seconds :)